airflow/providers/amazon/aws/auth_manager/constants.py

```diff
@@ -18,6 +18,8 @@
 # Configuration keys
 from __future__ import annotations
 
+CONF_ENABLE_KEY = "enable"
 CONF_SECTION_NAME = "aws_auth_manager"
+CONF_CONN_ID_KEY = "conn_id"
```

Comment:
Nit: Maybe give this a less generic name? I know it's under a section that makes it unique, but I think it's helpful to give it a more specific name. `aws_auth_mngr_conn_id`, `aws_auth_conn_id`, etc?

Reply:
I am not strongly opinionated on this one but I feel it will duplicate the section, don't you think? If I name the config `aws_auth_conn_id`, then the whole config will be `AIRFLOW__AWS_AUTH_MANAGER__AWS_AUTH_CONN_ID`. I really think the section is made for this: making the difference between similar fields between different component. But again, open to rename it if you still think this is a good idea